### ISSUE TITLE
The word 'Feiertag' throws an error

### ISSUE THREAD

**Issue by radomirbosak** [radomirbosak/duden-down#13]
To reproduce to bug:
```
$ ./duden.py Feiertag
Feiertag, der
=============
Word type: Substantiv, maskulin
Commonness: 3/5
Separation: Fei|er|tag
Traceback (most recent call last):
  File "./duden.py", line 202, in <module>
    main()
  File "./duden.py", line 198, in main
    word.describe()
  File "./duden.py", line 65, in describe
    if self.meaning_overview:
  File "./duden.py", line 176, in meaning_overview
    if entry.section and entry.section.h3.text == 'Beispiele':
AttributeError: 'NoneType' object has no attribute 'section'
```
version: v0.4.0

**Comment by radomirbosak** [comment]
Fixed in branch [develop](https://github.com/radomirbosak/duden-down/commits/develop).

**Comment by radomirbosak** [comment]
Fixed in [0.5.0](https://github.com/radomirbosak/duden-down/releases/tag/v0.5.0)
